pkg/analyser/analysis.go

```diff
@@ -51,13 +52,14 @@ type serializableDifference struct {
 }
 
 type serializableAnalysis struct {
-	Summary     Summary                                `json:"summary"`
-	Managed     []resource.SerializableResource        `json:"managed"`
-	Unmanaged   []resource.SerializableResource        `json:"unmanaged"`
-	Deleted     []resource.SerializableResource        `json:"missing"`
-	Differences []serializableDifference               `json:"differences"`
-	Coverage    int                                    `json:"coverage"`
-	Alerts      map[string][]alerter.SerializableAlert `json:"alerts"`
+	Summary         Summary                                `json:"summary"`
+	Managed         []resource.SerializableResource        `json:"managed"`
+	Unmanaged       []resource.SerializableResource        `json:"unmanaged"`
+	Deleted         []resource.SerializableResource        `json:"missing"`
+	Differences     []serializableDifference               `json:"differences"`
+	Coverage        int                                    `json:"coverage"`
+	Alerts          map[string][]alerter.SerializableAlert `json:"alerts"`
+	ProviderVersion string
```

Comment:
json struct tag is missing

Reply:
> You forgot to update HTML testdata, you can do it using the -update flag like for goldenfiles

I was waiting for the correct version of the html made by William